rjfoley/style.css

```diff
@@ -0,0 +1,23 @@
+/* Import fonts */
+
+/* Re-styling Basics i.e.: body h1 h2 h3 ... ; */
+
+/* Snippet Title */
+
+.only-text {
+	font-size:24px;
```

Comment:
This is nit picky as well, but you might consider consistently using ems, rems, pixels, or percents for font sizes, line height, padding, etc. For accessibility and responsive type, I'd recommend using ems or rems when possible

Reply:
ems rulez 😸 !  px is a deceptive and troublesome metric in the modern world of high density displays and widely varying display sizes.

ems and rems are inherently relative measures and I think of them as more flexible and future-proof.